topics/valkey.conf.md

```diff
@@ -34,6 +34,8 @@ The list of configuration directives, and their meaning and intended usage
 is available in the self documented example valkey.conf shipped into the
 Valkey distribution.
 
+* The self documented [valkey.conf for Valkey OSS 8.1](https://raw.githubusercontent.com/valkey-io/valkey/8.1/valkey.conf).
+* The self documented [valkey.conf for Valkey OSS 8.0](https://raw.githubusercontent.com/valkey-io/valkey/8.0/valkey.conf).
 * The self documented [valkey.conf for Valkey OSS 7.2](https://raw.githubusercontent.com/valkey-io/valkey/7.2/valkey.conf).
```

Comment:
It's just called Valkey.

For Redis, we had to add "OSS" according to their trademark policy.

Btw, we should probably remove the links to redis by now, but we can do that when we release 9.0.

```suggestion
* The self documented [valkey.conf for Valkey 8.1](https://raw.githubusercontent.com/valkey-io/valkey/8.1/valkey.conf).
* The self documented [valkey.conf for Valkey 8.0](https://raw.githubusercontent.com/valkey-io/valkey/8.0/valkey.conf).
* The self documented [valkey.conf for Valkey 7.2](https://raw.githubusercontent.com/valkey-io/valkey/7.2/valkey.conf).
```